documents/tutorials/contiv-policy.html

```diff
@@ -3,14 +3,14 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="Container Networking Tutorial">
+    <meta name="description" content="Advanced Contiv Tutorial">
```

Comment:
Why not just call it Contiv Policy Networking or Contiv Bandwidth Policy? Advanced is scarier and less searchable.